Uni_life_tracker/app/src/main/res/layout/activity_add_expenses.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think that it would be better to use LinearLayout instead of constraint because it has a cleaner look

Reply:
Sure thing